### ISSUE TITLE
[14.0] stock_available not saving settings in inventory -> settings -> Stock available to promise 

### ISSUE THREAD

**Issue by bluezebra-io** [OCA/stock-logistics-warehouse#1112]
![New - Odoo 2021-03-24 11-34-42](https://user-images.githubusercontent.com/69618711/112296274-08c3d680-8c95-11eb-8295-498bdd797d4f.png)

**Comment by hareshkansara** [comment]
Hello Sir,

Based on your issue post, i have carefully checked and found it’s due to dependency missing. Let me explain how it happens.

When you go to Inventory > Configuration > Settings, In section “Stock available to promise”

1. When you click on checkbox “Exclude incoming goods”   then it will install module named as stock_available_immediately (Ignore planned receptions in quantity available to promise), which is available in Odoo Version 14, so you have to also add this module in your addon list. Otherwise it will not work checkbox store.

Currently it’s working perfect in my local system, as i have clone this repository in my local system 
(https://github.com/OCA/stock-logistics-warehouse/tree/14.0)

2. When you click on checkbox “Include the production potential”  then it will install module named as stock_available_mrp (Consider the production potential is available to promise), but this module is not available in Odoo Version 14. So this checkbox will not work. This module is only available in V12.

So basically all these two checkbox feature is to install module when you tick checkbox and save.

Correct me if i understood something wrong.

**Comment by pedrobaeza** [comment]
`stock_available_mrp` is not yet migrated to 14.0, but the button tries to install, so it's not a bug, but not yet finished. Closing the issue.

**Comment by pedrobaeza** [comment]
@hareshkansara thanks for investigating it.
